curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fa537114e412950b62089.md

```diff
@@ -0,0 +1,136 @@
+---
+id: 671fa537114e412950b62089
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+In your JavaScript file, select the `.story-container` `div` and store it in a variable called `storyContainer`.
+
+# --hints--
+
+Your `storyContainer` variable should have the value of `document.querySelector(".story-container")`.
```

Comment:
We shouldn't give them the exact code here.

We should rephrase this to hint that they will need to access the DOM using the querySelector method.
Also, there should be a test before this to check that they created the variable first.

Reply:
Done!